### DISCUSSION TITLE
How to: Input Editable => Textarea Editable

### DISCUSSION THREAD

**Discussion by akno77** [pimcore/pimcore#12837] [Q&A / Support]
Is there a simple way to convert existing Input Editable data into Textarea Editable data?

Sometimes it can happen that an input has to be changed to a textarea editable but not losing the data.

**Comment by skynet-technologies** [comment]
You can use the textarea editor configuration functionalities of Pimcore. Please check the below reference URL.
https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Editables/Textarea.html

Hope it helps :)

**Comment by dvesh3** [comment]
you can write a simple script to do this:
```php
public function migrateEditableData(EditableLoaderInterface $editableLoader)
{
    $doc = Document\Page::getById(201);
    $inputEditable = $doc->getEditable('input1');
    $textAreaEditable = $editableLoader->build('textarea'); 
    $textAreaEditable->setName('textarea1'); //textarea editable should be present with the same name in the template
    $textAreaEditable->setDataFromResource($inputEditable->getData());
    $doc->setEditable($textAreaEditable);
    $doc->save();
 }
```
then get rid of input editable.
